core/arch/arm/kernel/thread_spmc.c

```diff
@@ -86,7 +87,7 @@ struct mem_frag_state {
 	SLIST_ENTRY(mem_frag_state) link;
 };
 
-static uint16_t my_sp_id = 0x8001;
+static uint16_t spmc_id = 0x8001;
```

Comment:
In `core/arch/arm/kernel/secure_partition.c` there's a `MIN_ENDPOINT_ID`. That should be a proper define (with a suitable name) in a .h file and we could skip this.

Reply:
I added it to thread_spmc.h. I will also need thread_spmc.h in the future. For example spmc_sp_handler will need to call the buffer functions inside thread_spmc.c